torch/csrc/jit/python/init.cpp

```diff
@@ -213,6 +213,8 @@ void initJITBindings(PyObject* module) {
       .def("_jit_pass_dedup_module_uses", &DedupModuleUses)
       .def("_jit_pass_replicate_dequantize", &ReplicateDeQuant)
       .def("_jit_pass_swap_dequantize", &SwapDeQuant)
+      .def("_jit_pass_swap_functional_linear", &SwapFunctionalLinear)
+      .def("_jit_pass_swap_functional_linear_in_module", &SwapFunctionalLinearInModule)
```

Comment:
I assume this pass is invoked prior to quant-fusion. Can you also make a change to convert_script to include this pass?

Reply:
This is called in Finalize, see future PRs in the stack